### ISSUE TITLE
Update to worked with opal-rails 0.9.2

### ISSUE THREAD

**Issue by zw963** [fkchang/opal-irb#16]
opal-rails 0.9.2 released, which Added support for sprockets-rails v3.

current opal-irb need old version to start.

**Comment by zw963** [comment]
Following is error messge:

```sh
Bundler could not find compatible versions for gem "opal":
  In snapshot (Gemfile.lock):
    opal (= 0.10.3)

  In Gemfile:
    opal-rails was resolved to 0.9.2, which depends on
      opal-activesupport (>= 0.0.5) was resolved to 0.3.0, which depends on
        opal (< 1.0.0, >= 0.5.0)

    opal-irb was resolved to 0.8.1, which depends on
      opal (~> 0.8.0)

    opal-irb was resolved to 0.8.1, which depends on
      opal-jquery (~> 0.4.0) was resolved to 0.4.2, which depends on
        opal (< 0.11.0, >= 0.7.0)

    opal-rails was resolved to 0.9.2, which depends on
      opal (< 0.11, >= 0.8.0)

    opal-rails was resolved to 0.9.2, which depends on
      opal-sprockets (~> 0.4.0) was resolved to 0.4.0.0.10.0.3.0.0, which depends on
        opal (~> 0.10.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

```

**Comment by fkchang** [comment]
Hi @zw963 - I'll try to get to this as soon as possible

**Comment by zw963** [comment]
Thanks, maybe I could help you about refactor some ruby code.

compare to ruby community style, this project is a little specially ... : )

**Comment by fkchang** [comment]
@zw963 it would be appreciated, opal-irb is due for a lot of maintenance, I just haven't been able to find time

**Comment by zw963** [comment]
@fkchang , I try to create a PR to refine Ruby code.

It should be safe, could work as well as before.

**Comment by fkchang** [comment]
@zw963 wow, that's fast. Thanks. On rubocop, I'm generally fine w/many of the defaults. The one that don't agree with is that blocks should always be do end when > than one line. While I follow the '' vs "" rule, I don't particularly like it, I've been bitten in changing strings to have interpolation or not, to think it doesn't really bring value to me

**Comment by zw963** [comment]
@fkchang , 

>> he one that don't agree with is that blocks should always be do end when > than one line.

Fine,  I agree a lot, especially, write many js forks. (though, I write js less)

>>  I don't particularly like it, I've been bitten in changing strings to have interpolation or not, to think it doesn't really bring value to me

Because `opal-irb` support emacs binding, so I think you use emacs? with some library, change this
, just one hotkey, I binding to ``Ctrl+'``, If you don't like , we can keep no changed, tough,  escape things some a little anoise ...

BTW: If you don't like some config, you can create `~/.rubocop`, to disabled or change it default value.

**Comment by zw963** [comment]
Oh, for this issue, I have make `opal-irb` worked with my Rails 5.0.2 project production with some simple hack, so, i think just some gemspec to update dependency version things.

**Comment by fkchang** [comment]
@zw963 yeah, I have the quotes changing binding, I accept rubocop suggestion because it's easy to toggle in emacs, but I have been bitten in that I forgot to change it back to double quotes and then don't get the interpolation desired.  Again, I think I prepare consistency here, don't think, just always do "", but I've gotten used to it.  I should make a project .rubocop

**Comment by fkchang** [comment]
@zw963 Re: gettting it to work w/Rails 5.0.2 -it's likely that is all that's needed, what I'll end up doing is changing the gemspec and then testing. Unfortunately, I need more tests, and some things I could not figure out how to test (like require_js) short of trying by hand. I guess I could add some selenium type tests.  If lucky, it's just a gemspec update, I have needed to change code between opal versions

**Comment by zw963** [comment]
Oops, this repo never updated since 2017 ...
